tsconfig.build.json

```diff
@@ -23,6 +23,12 @@
     "declaration": true,
     "declarationMap": true,
     "incremental": true,
-    "preserveWatchOutput": true
+    "preserveWatchOutput": true,
+
+
+    // There are two duplicate type definitions included from `chai` and `vitest` packages.
```

Comment:
i don't think comments like this work in json

Reply:
Typescript use `json5` standard for configuration which have support for comments. 

https://json5.org/